README.rst

```diff
@@ -19,34 +19,52 @@ from the protobuf source IDL and additional configuration in YAML files.
 Installation
 ------------
 
-At the moment, this library is under development, so please see
-`using a development checkout`_ for installation instructions
+You can now install artman directly from pip:
 
-.. _`using a development checkout`: https://github.com/googleapis/artman/blob/master/CONTRIBUTING.rst#using-a-development-checkout
+.. code::
+
+    pip install artman
```

Comment:
Yes. Thank you -- fixing. (Also, it will not be on PyPI until after this merge goes in.)